libs/libc/misc/lib_cxx_initialize.c

```diff
@@ -71,11 +71,11 @@ void lib_cxx_initialize(void)
 #else
       initializer_t *initp;
 
-      sinfo("_sinit: %p _einit: %p\n", &_sinit, &_einit);
+      sinfo("_sinit: %p _einit: %p\n", _sinit, _einit);
 
       /* Visit each entry in the initialization table */
 
-      for (initp = &_sinit; initp != &_einit; initp++)
+      for (initp = _sinit; initp != _einit; initp++)
```

Comment:
```suggestion
      for (initp = _sinit; initp < _einit; initp++)
```
should not happen, but just for safety reasons

Reply:
Done.